addons/binding/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/handler/HarmonyHubHandler.java

```diff
@@ -81,9 +92,7 @@
 
     private HarmonyClient client;
 
-    private HarmonyConfig cachedConfig;
-
-    private Date cacheConfigExpireDate;
+    private ExpiringCacheAsync<HarmonyConfig> configCache = new ExpiringCacheAsync<>(CONFIG_CACHE_TIME);
```

Comment:
What would you think about using the following concept, I think that it reads better especially with big numbers.

```java
TimeUnit.MINUTES.toMillis(1);
```

Or as the constant of course

Reply:
Replaced.